src/pages/LoginPage/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './LoginPage';
```

Comment:
```suggestion
import React from 'react';

import LoginForm from '@src/components/LoginForm';

function LoginPage() {
  return (
    <div>
      로그인
      <LoginForm />
    </div>
  );
}

export default LoginPage;
```

index.ts를 이렇게 가져가는 게 낫지 않을까.. ?

Reply:
@sewonkimm 일다는 index에 import만 모는 거스로